### ISSUE TITLE
USH-011 — Length of Password is Not Validated After Reset

### ISSUE THREAD

**Issue by willdoran** [ushahidi/platform#1606]
### Expected behaviour
- Use Server-Side Validation of the length and complexity of the password.
### Actual behaviour
- When using the password protection link it is possible to alter to request and create a password that is only one character long or when you intercept the login request an user-account without password.

**Comment by rowasc** [comment]
@willdoran is this in the right column?

**Comment by willdoran** [comment]
Yeah I think this is done

**Comment by rowasc** [comment]
@willdoran are we waiting on lumen to deploy it?

**Comment by willdoran** [comment]
@rowasc Yep it was merged into Lumen.

**Comment by rjmackay** [comment]
Dropped back to QA to test the rebased version

**Comment by rowasc** [comment]
Testing

**Comment by rowasc** [comment]
Password is validated in the backend. 
Passes

**Comment by Eve-wanderi** [comment]
Also confirme dthis from my end Password is Validated.
